pkg/portal/portal.go

```diff
@@ -44,6 +44,10 @@ type Runnable interface {
 	Run(context.Context) error
 }
 
+const (
+	featureFlags = "featureflags"
```

Comment:
nit: I think extracting this to a constant is unnecessary for its one use below

Reply:
I think so, too. Reverting the change suggested by Aldo.

Reply:
Done!